test-app/package.json

```diff
@@ -0,0 +1,137 @@
+{
+  "name": "test-app",
```

Comment:
It seems to be missing `private: true`. That might be helpful to prevent mistakes.